tests/nnfw_api/src/one_op_tests/Conv2D.cc

```diff
@@ -35,7 +35,7 @@ TEST_F(GenModelTest, OneOp_Conv2D)
   _context->addTestCase(uniformTCD<float>(
     {{4, 0, -5, 1, 0, 4, -1, 1, -1, -3, 3, -2, -4, 1, -2, 2, 4, -4, 2, 2, 0, 4, -1, -2, 4}},
     {{47, -4, -25, 9, 10, 10, -13, 11, -14, -26, -12, 26, 20, 40, 1, 3, 11, 4}}));
-  _context->setBackends({"acl_cl", "acl_neon", "cpu", "ruy", "xnnpack"});
+  _context->setBackends({"acl_cl", "acl_neon", "cpu", "ruy", "xnnpack", "gpu_cl"});
```

Comment:
please add neg tc.

Reply:
Such like neg_OneOp_Conv2D_Type testcases already exist. 
And think that was running for everybackend.